gst-libav.yaml

```diff
@@ -0,0 +1,104 @@
+package:
+  name: gst-libav
+  version: "1.26.0"
+  epoch: 0
+  description: GStreamer plugin for using the encoders, decoders, muxers, and demuxers provided by FFmpeg.
+  copyright:
+    - license: LGPL-2.0-or-later
+
+environment:
+  contents:
+    packages:
+      - build-base
+      - cmake
+      - ffmpeg-dev
+      - gst-plugins-base-dev
+      - gstreamer-dev
+      - meson
+      - python3
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/GStreamer/gstreamer
+      tag: ${{package.version}}
+      expected-commit: d31ce8e5e1aacf8f5e5beabb5c81ce2e4da5c202
+
+  - runs: |
+      cd subprojects/gst-libav
+      CFLAGS="$CFLAGS -O2" \
+      CXXFLAGS="$CXXFLAGS -O2" \
+      CPPFLAGS="$CPPFLAGS -O2" \
+      meson \
+        -Db_lto=true \
+        -Dtests=enabled \
+        -Dpackage-name="gst-libav (Wolfi)" \
+        -Dpackage-origin="https://wolfi.dev" \
+        --prefix=/usr \
+        . output
+      meson compile -C output
+
+      DESTDIR="${{targets.destdir}}" meson install --no-rebuild -C output
+
+  - uses: strip
+
+subpackages:
+  - name: ${{package.name}}-dev
+    pipeline:
+      - uses: split/dev
+    description: ${{package.name}} dev
+
+  - name: ${{package.name}}-doc
+    pipeline:
+      - uses: split/manpages
+    description: ${{package.name}} manpages
+
+update:
+  enabled: true
+  github:
+    identifier: GStreamer/gstreamer
+    use-tag: true
+    tag-filter-prefix: "1."
+
+test:
+  environment:
+    contents:
+      packages:
+        - git
+  pipeline:
+    - uses: test/ldd-check
+      with:
+        packages: ${{package.name}}
```

Comment:
we're using `test/tw/ldd-check` more and more, this can update or I've also suggested moving this only for `-dev` packages but we can keep it here as well.